bugbug/models/qaneeded.py

```diff
@@ -96,6 +96,13 @@ def get_labels(self):
                             for label in ["qawanted", "qe-verify", "qaurgent"]
                         ):
                             classes[bug_id] = 1
+
+                        if any(
+                            change["removed"].startswith(label)
+                            for label in ["qawanted", "qe-verify", "qaurgent"]
+                        ):
+                            classes[bug_id] = 0
```

Comment:
Move it *before* the other if which sets the class to 1.
This way for example if qawanted is removed but qe-verify added, we will still have the class be 1.

Reply:
oh okay ...